ADMINISTRATORS.md

```diff
@@ -0,0 +1,59 @@
+# Dapr administrators
+
+## Charter
+
+Administrators have access to credentials equivalent to members of the steering and technical commitee (STC) and are responsible to perform pre-approved routine maintenance duties and ad-hoc tasks delegated by the STC without making isolated decisions. This document outlines a list of pre-approved duties and some that require STC approval - when in doubt, the administrator should seek guidance from one STC member.
+
+### Pre-approved duties
+
+An administrator might perform the following duties without requiring approval from any STC member:
```

Comment:
Given these duties, this seems like there needs to be a new administrators group in the github that has these privileges. Similar to the release team account. If so, can you add into the steps below that on approval by STC, administrators are added to this account? This is in addition to having access to the STC vault in 1Password for credentials.

Reply:
I agree.

Reply:
I didn't see any update below on this comment. I suggest creating a github team called `administrators` here https://github.com/orgs/dapr/teams and then saying that this is what makes you and admin as well as access to the STC 1password vault.

Reply:
I am adding the text now. GitHub does not allow a Team to be made org owner, only individuals. So, we cannot really create an `administrators` role.